_includes/pbc/all/install-features/202400.0/install-the-product-offer-service-points-feature.md

```diff
@@ -567,3 +569,31 @@ class ProductOfferStorageDependencyProvider extends SprykerProductOfferStorageDe
 }
 ```
 
+**src/Pyz/Zed/ProductOfferGui/ProductOfferGuiDependencyProvider.php**
+
+```php
+<?php
+
+namespace Pyz\Zed\ProductOfferGui;
+
+use Spryker\Zed\ProductOfferServicePointGui\Communication\Plugin\ProductOfferGui\ServiceProductOfferViewSectionPlugin;
+
+class ProductOfferGuiDependencyProvider extends SprykerProductOfferGuiDependencyProvider
+{
+    /**
+     * @return array<\Spryker\Zed\ProductOfferGuiExtension\Dependency\Plugin\ProductOfferViewSectionPluginInterface>
+     */
+    public function getProductOfferViewSectionPlugins(): array
+    {
+        return [
+            new ServiceProductOfferViewSectionPlugin(),
+        ];
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
```

Comment:
Let's extend the verification: 

Log in to the backoffice.
Navigate to the "Marketplace" >> "Offers" section in the navigation menu.
Select the specific product offer you want to view.